src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/EditIssuePage.kt

```diff
@@ -31,7 +34,8 @@ class EditIssuePage(
 
     fun fillForm(): EditIssuePage {
         summaryLocator.clearAndTypeIfPresent("summary")
-        descriptionLocator.clearAndTypeIfPresent("description")
+        //By.id("description").clearAndTypeIfPresent("Test Desc")
```

Comment:
We should keep the codebase clean of dead/commented out code

Reply:
sure, removed